src/ui/src/js/controllers/command_index.js

```diff
@@ -29,12 +29,18 @@ export default function commandIndexController(
   detailSystems,
   commands) {
   $scope.setWindowTitle('commands');
-
+  $scope.filterHidden = false;
+  $scope.dtInstance={};
   $scope.dtOptions = DTOptionsBuilder.newOptions()
     .withOption('order', [4, 'asc'])
     .withOption('autoWidth', false)
     .withBootstrap();
 
+
+  $scope.hiddenComparator = function(expected, actual){
+    return actual || !expected;
+  };
```

Comment:
These names don't really tell me what's going on here. I'm assuming that this function is getting called once per item that's being filtered? But I don't understand what would be "expected" or "actual" in that context.